feat(elasticloadbalancing): classic load balancer supports ec2 instances (#24353)
aws/aws-cdk: packages/@aws-cdk/aws-elasticloadbalancing/README.md

  allowConnectionsFrom: [mySecurityGroup],
});
```

### Adding Ec2 Instance as a target for the load balancer

> EC2 instaces can be added as the target for the load balancer via `addTarget()` method using `InstanceTarget` class by providing ec2 `instance` as a target.

Reviewer: ```suggestion
You can add an EC2 instance to the load balancer by calling using `new InstanceTarget` as the argument to `addTarget()`:
```

```ts
const lb = new elb.LoadBalancer(this, 'LB', {
  vpc,
});
// instance to add as the target for load balancer.
const instance = new Instance(stack, 'targetInstance', {
  vpc: vpc,
  instanceType: InstanceType.of(InstanceClass.BURSTABLE2, InstanceSize.MICRO),
  machineImage: new AmazonLinuxImage(),
});

> lb.addTarget(elb.InstanceTarget(instance))

Reviewer: ```suggestion
lb.addTarget(elb.InstanceTarget(instance));
```
